.github/workflows/ports_psoc6.yml

```diff
@@ -79,7 +79,18 @@ jobs:
         echo " >> PSoC6 single board tests"
         ./psoc6/run_psoc6_tests.sh -c --psoc6 --dev0 ${devs[0]} 
         echo " >> PSoC6 extended hardware tests"
-        ./psoc6/run_psoc6_tests.sh -c --psoc6-hwext --dev0 ${devs0[0]} 
+        ./psoc6/run_psoc6_tests.sh -c --psoc6-hwext --dev0 ${devs0[0]}
+        echo " >> PSoC6 i2c tests"
+        if [ "${{ matrix.board }}" == "CY8CPROTO-062-4343W" ]; then
```

Comment:
By the way, this dirty selection approach (sorry for that 😞)  should be tackle in the rework of this run_psoc6_tests.sh in future. I might have the chance to start as pre-1.0.0.